src/content/en/tools/workbox/modules/workbox-strategies.md

```diff
@@ -192,6 +193,109 @@ registerRoute(
+
+* `handle()`: Perform a request strategy and return a `Promise` that will resolve with a `Response`,
+  invoking all [relevant plugin
+  callbacks](/web/tools/workbox/guides/using-plugins#lifecycle_callbacks).
+* `handleAll()`: Similar to `handle()`, but returns two `Promise` objects. The first is
+  equivalent to what `handle()` returns and the second will resolve when promises that were
+  added to `event.waitUntil()` within the strategy have completed.
+
+Both request strategies are invoked with two parameters:
+
+- `request`: The `Request` the strategy will return a response for.
+- `handler`: A `StrategyHandler` instance automatically created for the current strategy.
+
+### Creating A New Strategy
+
+The following is an example of a new strategy that re-implements the behavior of `NetworkOnly`:
+
+```javascript
+class NewNetworkOnlyStrategy extends Strategy {
+  _handle(request, handler) {
+    return handler.fetch(request);
+  }
+}
+```
+
+Notice how `handler.fetch()` is called instead of the native `fetch` method. The `StrategyHandler`
+class provides a number of fetch and cache actions that can be used whenever `handle()` or
+`handleAll()` is used:
+
+* `fetch`: Fetches a given request, and invokes the `requestWillFetch()`, `fetchDidSucceed()`, and
+  `fetchDidFail()` plugin lifecycle methods
+* `cacheMatch`: Matches a request from the cache, and invokes the `cacheKeyWillByUsed()` and
+  `cachedResponseWillByUsed()` plugin lifecycle methods
+* `cachePut`: Puts a request/response pair in the cache, and invokes the `cacheKeyWillByUsed()`,
+  `cacheWillUpdate()`, and `cacheDidUpdate()` plugin lifecycle methods 
+* `fetchAndCachePut`: Calls `fetch()` and runs `cachePut()` in the background on the response
+  generated by `fetch()`.
+* `hasCallback`: Takes a callback as input and returns true if the strategy has at least one plugin
+  with the given callback.
+* `runCallbacks`: Runs all plugin callbacks matching a given name, in order, passing a given param
+  object (merged with the current plugin state) as the only argument.
+* `iterateCallbacks`: Accepts a callback and returns an iterable of matching plugin callbacks, where
+  each callback is wrapped with the current handler state (i.e. when you call each callback,
+  whatever object parameter you pass it will be merged with the plugin's current state).
+* `waitUntil`: Adds a promise to the extend lifetime promises of the event event associated with the
+  request being handled (usually a `FetchEvent`).
+* `doneWaiting`: Returns a promise that resolves once all promises passed to `waitUntil()` have
+  settled.
+* `destroy`: Stops running the strategy and immediately resolves any pending `waitUntil()` promises.
+
+Note: Refer to the [source
+implementation](https://github.com/GoogleChrome/workbox/blob/6d38919ebbc9664327e19ff00302d805c8166170/packages/workbox-strategies/src/StrategyHandler.ts)
+of `StrategyHandler` to see all the parameters accepted by each action
```

Comment:
Was thinking of showing code examples of each action, but thought that might be overkill?

Let me know if this works!